cairo-listings/src/main.rs

```diff
@@ -187,7 +239,58 @@ fn process_file(manifest_path: &str, args: &VerifyArgs) {
     // FORMAT CHECKS
     if !tags.contains(&Tags::IgnoreFormat) && !args.formats_skip {
         // This program must pass cairo-format
-        let _ = run_command(ScarbCmd::Format(), manifest_path, file_path, vec![]);
+        let mut format_args = vec![];
+
+        // Add the -c flag for verification
+        format_args.push("-c".to_string());
```

Comment:
this can be done in a single line

```suggestion
        let mut format_args = vec!["-c".to_string()];
```